KthLargestElementInAnArray.java

```diff
@@ -0,0 +1,25 @@
+
+// TC : O(n*logn + k*logn)
```

Comment:
Kindly check your complexity , if its nlogn then there is no point  of using heaps